test-projects/social-login/setupAuthRoutes.js

```diff
@@ -0,0 +1,98 @@
+const { startAuthedSession } = require('@keystone-alpha/session');
+
+module.exports = ({
+  strategy,
+  authRoot = '/auth',
+  successRedirect = '/api/session',
+  failureRedirect = '/',
+}) => {
+  // Hit this route to start the auth process for service
```

Comment:
Adding `const basePath = `${authRoot}/${strategy.authType}`;` here will allow a bunch of code to be simplified in this file and make it more clear where the other aspects of the `strategy` object are being used.